wpeverest/everest-forms
# Enhance - Added navigation design for tools page

### All Submissions:






### Changes proposed in this Pull Request:


Added navigation design for Tools pages.

### How to test the changes in this Pull Request:

1. Go to Tools Pages
2. See if design is working good.

### Types of changes:




### Other information:




### Changelog entry
> Enhance - Added navigation design for tools page
